core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -40,13 +40,28 @@ oppia.directive('topNavigationBar', [
           $scope.NAV_MODE = GLOBALS.NAV_MODE;
           $scope.LABEL_FOR_CLEARING_FOCUS = LABEL_FOR_CLEARING_FOCUS;
           $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
-
+          
+          $scope.activeMenu = '';
           $scope.username = GLOBALS.username;
           $scope.profilePictureDataUrl = GLOBALS.profilePictureDataUrl;
           $scope.isAdmin = GLOBALS.isAdmin;
           $scope.isModerator = GLOBALS.isModerator;
           $scope.isSuperAdmin = GLOBALS.isSuperAdmin;
           $scope.logoutUrl = GLOBALS.logoutUrl;
+          $scope.KEYBOARD_EVENT_TO_KEY_CODES = {
+            enter: {
+              isShiftKeyPressed: false,
```

Comment:
Actually let's make this shiftKeyIsPressed to make it clear it's a boolean and not a function (i.e. make it not start with a verb).